test/test.authclient.ts

```diff
@@ -38,4 +39,31 @@ describe('AuthClient', () => {
       assert.equal(authClient[camelCased], value);
     }
   });
+
+  it('should allow disabling of the default interceptor', () => {
```

Comment:
Ultimately, I'm a bit confused about this change: are we replacing functionality or removing it entirely? If replacing, it would be nice to see more test coverage (something at parity with confirming transporters functionality). If the latter, then I think maybe it would be best to add some type of warning for users attempting to use the Transporter property.